packages/target-tools/src/telemetryProvider.spec.js

```diff
@@ -111,56 +121,105 @@ describe("TelemetryProvider", () => {
       })
     );
   });
-
-  it("assigns local execution mode", () => {
+  it("assigns edge execution mode for server side", () => {
     const provider = TelemetryProvider();
 
+    const DELIVERY_RESPONSE_SERVER_SIDE = {
```

Comment:
Let's reuse the DELIVERY_RESPONSE you've defined earlier.  You can use use the assign() function to create a copy of the object and then modify specific fields if you need to.

Reply:
Good idea 👍 .